sdc/tests/test_series.py

```diff
@@ -4779,7 +4779,8 @@ def test_impl(S, ind, val):
 
         with self.assertRaises(TypingError) as raises:
             hpat_func(S, ind2, value2)
-        msg = 'Operator setitem. The object idx' + '\n' + ' given: unicode_type' + '\n' + ' expected: int, Slice, Series'
+        msg = 'Operator setitem. The object idx' + '\n' + ' given: unicode_type' + '\n' \
```

Comment:
Why not avoid concatenation of string via `+`?
`msg = 'Operator setitem. The object idx\n given: unicode_type\n expected: int, Slice, Series'`